src/api/consumer/lib/commit.js

```diff
@@ -117,6 +118,12 @@ export async function commitAllAction(args: {
 
   commitResults.newComponents = newComponents;
   HooksManagerInstance.triggerHook(POST_TAG_ALL_HOOK, commitResults);
+  Analytics.setExtraData(
+    'num_components',
+    commitResults.taggedComponents.length +
```

Comment:
are you sure everything is number? Might it be null or undefined maybe?
Also, it cost nothing to put each one of them in a different key. lets separate them